swagger.yaml

```diff
@@ -117,6 +117,52 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /v1/patients/{patientId}/exists:
```

Comment:
Do not use method names in URLs, appropriate way to only check existance of resource would be to use HEAD -verb instead of GET and use the same path as GET 

e.g. 

    /v1/patients/{patientId}:
      head: